### PR TITLE
FreeBSD build fix, adding proper dependency on backtrace too.

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

File: folly/portability/Config.h [hunk 1]
```diff
@@ -37,3 +37,7 @@
 #include <AvailabilityMacros.h> // @manual
 #include <TargetConditionals.h> // @manual
 #endif
+
+#ifdef __FreeBSD__
+#include <sys/types.h>
```
Comment (yfeldblum) [inline folly/portability/Config.h:42]: What is this include for?

File: folly/experimental/symbolizer/Dwarf.cpp [hunk 2]
```diff
@@ -1317,9 +1317,11 @@ Dwarf::LineNumberVM::StepResult Dwarf::LineNumberVM::step(
       // use unbounded amounts of state (ie. use the heap).  We'll do a second
       // pass (using nextDefineFile()) if necessary.
       break;
+#if !defined(__FreeBSD__)
     case DW_LNE_set_discriminator:
```
Comment (yfeldblum) [inline folly/experimental/symbolizer/Dwarf.cpp:1321]: Does freebsd not have this constant?

Comment (devnexen) [reply folly/experimental/symbolizer/Dwarf.cpp:1321]: it does not.

File: folly/portability/Config.h (hunk 1 above)
Comment (devnexen) [reply folly/portability/Config.h:42]: pid_t/fork_t

Comment (yfeldblum) [reply folly/portability/Config.h:42]: Let's move this include to those files which need it.

File: folly/detail/AtFork.h [hunk 3]
```diff
@@ -18,6 +18,10 @@
 
 #include <folly/Function.h>
 
+#if defined(__FreeBSD__)
+#include <sys/types.h>
+#endif
+
 namespace folly {
 
 namespace detail {
```
Comment (yfeldblum) [inline folly/detail/AtFork.h:27]: I think `sys/types.h` is portable? Also, folly conventional include order.

```
#include <sys/types.h>

#include <folly/Function.h>
```

**Pull request closed without merging**